lib/dor/was_seed/thumbnail_generator_service.rb

```diff
@@ -7,32 +7,29 @@ class ThumbnailGeneratorService
       # because this date is earlier than any of the archived dates of the content,
       # this tells openwayback to provide the earliest capture date.
       DATE_TO_TRIGGER_EARLIEST_CAPTURE_DATE = '19900101120000'.freeze
-      def self.capture_thumbnail(druid, workspace, uri)
-        thumbnail_file = "#{DruidTools::Druid.new(druid, workspace).content_dir}/thumbnail.jp2"
-        wayback_uri    = "#{Settings.was_seed.wayback_uri}/#{DATE_TO_TRIGGER_EARLIEST_CAPTURE_DATE}/#{uri}"
-        temporary_file = "tmp/#{druid[6, 14]}"
-        jpeg_file = "#{temporary_file}.jpeg"
+      def self.capture_thumbnail(druid, workspace, seed_uri)
+        screenshot_jpeg = "tmp/#{druid[6, 14]}.jpeg"
```

Comment:
What an odd way of saying `delete_prefix`.

Worth changing to be consistent with other codebases?

Reply:
done.